modules/rosa-list-objects.adoc

```diff
@@ -176,6 +175,42 @@ List all API and ingress endpoints for a cluster named `mycluster`:
 $ rosa list ingresses --cluster=mycluster
 ----
 
+== list instance-types
+
+List all of the available instance types for use with {product-title}. Availability is based on the account's AWS quota.
+
+.Syntax
+[source,terminal]
+----
+$ rosa list instance-types [arguments]
+----
+
+.Optional arguments inherited from parent commands
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--help
+|Shows help for this command.
+
+|--debug
+|Enables debug mode.
+
+|--output
+|The output format. Allowed formats are [json | yaml].
```

Comment:
Need to escale this pipe so that it doesn't wrap to a different row:
```suggestion
|The output format. Allowed formats are [json \| yaml].
```

Reply:
TY and fixed @vkareh